blocks/embed/embed.js

```diff
@@ -0,0 +1,156 @@
+const scPlayerHeight = window.hlx.aemassets?.videosmartcrop?.height || '44rem';
```

Comment:
Can't this be controlled via css only instead of introducing a property in code to handle it?